frontend/web/components/modals/CreateUser.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {Component, useEffect, useState} from 'react';
+import ChipInput from '../ChipInput';
+import ErrorMessage from '../ErrorMessage';
+import Button from "../base/forms/Button";
+import {FC} from 'react'
+import {useCreateIdentitiesMutation} from "common/services/useIdentity"; // we need this to make JSX compile
+import Utils from 'common/utils/utils'
+type CreateUserType = {
+    environmentId: string
+}
+
+const CreateUser: FC<CreateUserType> = ({environmentId}) => {
+    const [value, setValue] = useState<string[]>([]);
+    const [createIdentities, {isSuccess, isError} ] = useCreateIdentitiesMutation()
+    const submit = ()=> {
+        createIdentities({identifiers:value, environmentId, isEdge: Utils.getIsEdge()})
+    }
+    useEffect(()=>{
+        if(isSuccess) {
+            closeModal()
+        }
+    },[isSuccess])
+    return (
+        <div>
+            <FormGroup>
+                <label>
+                    User IDs
+                </label>
+            </FormGroup>
+            <FormGroup className="text-right">
+                <ChipInput
```

Comment:
While we're working on this, is it possible to update the text colour when using dark mode? 

<img width="614" alt="image" src="https://user-images.githubusercontent.com/14089968/220679661-7ca66335-7d90-4842-9380-70f3d988b002.png">

Reply:
Thanks for raising, will do

Reply:
Done 

![image](http://g.recordit.co/P8oDmSmC21.gif)

Reply:
Yep, nice, looks good.